uint16 id

string robot_name1
string robot_name2
uint16 id1
uint16 id2